src/components/Blinds/Blinds.jsx

```diff
@@ -0,0 +1,58 @@
+import React, {useState} from 'react';
+import { render } from 'react-dom';
+import './style.css';
+import blindsOpen from '../../assets/blinds-open.svg';
+import blindsClosed from '../../assets/blinds-closed.svg';
+
+export const Blinds = (props) => {
+
+    let initialOpened = false;
+    if(props.blinds == 'open')
+    {
+        console.log(props.blinds);
+        initialOpened = true;
+    }
+    else 
+    {
+        initialOpened = false;
+    }
+    const [opened, setOpened] = useState(initialOpened);
+
+    let upperClass = "button button--active";
+    let lowerClass = "button";
+    let image = blindsOpen;
+
+    const changeState = () =>
+    {
+        () => setOpened(!opened);
+    }
```

Comment:
nepoužívaná funkce? Navíc nic nedělá, mělo by být kdybysh jí chtěla použít
```
    const changeState = () =>
    {
     setOpened(!opened);
    }
```
ale jak to děláš napřímo teď je leopší 👍